apps/auction2/templates/index.jade

```diff
@@ -32,8 +32,9 @@ block body
               //- rendered on the client
             .cf-total-sort__sort
               //- rendered on the client
-          #cf-artworks
-            //- rendered on the client
-          hr
-          .cf-pagination
-            //- rendered on the client
+          .cf-bottom-right-part
+            #cf-artworks
+              //- rendered on the client
```

Comment:
Out of curiosity, what is this comment convention that I'm seeing throughout force as I dig around? What is it indicating exactly vs not being there?